test/cli/suggest-sig/suggest-sig.out

```diff
@@ -306,7 +306,7 @@ suggest-sig.rb:112: This function does not have a `sig` https://srb.help/7017
      112 |  def self.load_account_business_profile(merchant)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   Autocorrect: Done
-    suggest-sig.rb:112: Inserted `sig {params(merchant: T.untyped).returns(Integer)}
+    suggest-sig.rb:112: Inserted `sig {params(merchant: TestCarash::Merchant).returns(Integer)}
```

Comment:
This is a case where we weren't able to get information about the `merchant` parameter before, as it was used locally with a keyword args send.